src/_reducers/securityUsers.reducer.js

```diff
@@ -12,6 +12,22 @@ const {
   RESET_STATUS_MESSAGE
 } = constants;
 
+const getErrorMessage = (error) => {
+  if (error.hasOwnProperty('email')) {
+    return error.email[0].replace(/[[\]']/g, '');
```

Comment:
I take it that the backend team never fixed this weird error message. You can add it as a todo to nag them next year...2019 new year goal 🤣

Reply:
🤣 alright. I'll use the same todo you've left in the `assets.reducer`